website/config.toml

```diff
@@ -127,3 +127,16 @@ name = "GitHub"
 weight = 99
 url = "https://github.com/awslabs/karpenter"
 pre = "<i class='fab fa-github'></i>"
+
+# Add your release versions here
+[[params.versions]]
+  version = "latest"
+  url = "https://karpenter-versions.gcline.us/docs/"
+
+[[params.versions]]
+  version = "v0.4.3"
+  url = "https://karpenter-versions.gcline.us/docs/"
+
+[[params.versions]]
+  version = "Pre-release"
```

Comment:
```suggestion
  version = "preview"
```

Reply:
I'd rather keep this as is. Terms are equivalent. difficult to change. Knative and kubernetes use pre-release.

Reply:
Knative actually calls it `(Pre-release)` w/ path `development`. https://knative.dev/development/. Not a hill I want to die on.